### ISSUE TITLE
Interactive search filter UI tabbing behavior is odd

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#12917]
When tabbing through interactive search filter UI, edit mode activates, filters can change order, and tab order is inconsistent between Edgium and Safari. With VoiceOver on, the user can get into an infinite loop trying to navigate past the filters with the VoiceOver cursor. 

Example in Safari, VoiceOver off:

![2020-08-11 09 17 30](https://user-images.githubusercontent.com/206864/89922244-9f82cd80-dbb3-11ea-9d5d-7269499a9318.gif)

**Comment by poojaj-tech** [comment]
@limitedmage thanks for filing this - closing this issue as we have decided to remove interactive search. ([slack conversation](https://sourcegraph.slack.com/archives/CHEKCRWKV/p1600443200010300))
